src/components/ListItem.jsx

```diff
@@ -55,8 +74,13 @@ export function ListItem({ item, listPath }) {
 		return;
 	};
 
+	const UrgencyStatusIcon = urgencyStatusIcons[itemUrgencyStatus];
+
 	return (
-		<li className="ListItem">
+		<li className={`ListItem`}>
```

Comment:
Curious why this is a template literals for a static class name?

Reply:
great catch! It is now fixed, it was from past styling attempts